packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -732,6 +729,11 @@ export class IndexedDbPersistence implements Persistence {
     );
   }
 
+  static clearPersistence(persistenceKey: string): Promise<void> {
+    const dbName = persistenceKey + IndexedDbPersistence.MAIN_DATABASE;
+    return SimpleDb.delete(dbName);
```

Comment:
Prefer making this an async function and doing an `await` here to returning the promise directly.

I know this doesn't make much of a difference but makes it conceptually cleaner and easier to refactor later.

Basically we want to be in the habit of making things async and avoiding the Promises as much we can.